packages/cascara/src/ui/Form/modules/ActionEdit/ActionEdit.js

```diff
@@ -70,7 +70,7 @@ const ActionEdit = ({
       onAction(
         // fake target
         {
-          name: 'edit.save',
+          name: 'form.submit',
```

Comment:
This is a breaking change if we are changing the actions, right? Don't we need to make a migration path so that the old action names still work before we break them here? Maybe I am not understanding.

Reply:
This is not a breaking change, since it only applies to `Form` and `Form` is an alpha version.

Again, just an attempt to be more semantic in the naming convention. But happy to change it back if this is not what we want.

Reply:
@brianespinosa This change was applied to Form's dedicated `ActionEdit` module to match the Form semantics I was trying to set up. This has been changed back to its original version.

Reply:
👍